<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TPGPSClass</class>
 <widget class="QMainWindow" name="TPGPSClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TPGPS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="portStatus">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Port : </string>
    </property>
   </widget>
   <widget class="QLabel" name="Latitude">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Latitude: </string>
    </property>
   </widget>
   <widget class="QLabel" name="Longitude">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Longitude: </string>
    </property>
   </widget>
   <widget class="QLabel" name="TexteMemo1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MÃ©mo 1 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Memo1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Trame1</string>
    </property>
   </widget>
   <widget class="QLabel" name="MemoTexte2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Trame 2 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Memo2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Trame2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TPGPS.qrc"/>
 </resources>
 <connections/>
 <slots>
  <signal>readyRead()</signal>
  <slot>onButtonclicked()</slot>
  <slot>serialPortRead()</slot>
 </slots>
</ui>
